content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -613,6 +613,249 @@ To test it, change the context to `myuser`:
 kubectl config use-context myuser
 ```
 
+<!-- TODO this should become a task page -->
+## How to delegate signer permissions within a domain {#custom-domain-delegation}
+
+{{< feature-state feature_gate_name="ValidatingAdmissionPolicy" >}}
+
+When working with [custom signers](#custom-signers) that approve, deny and sign 
+[CertificateSigningRequests](/docs/reference/kubernetes-api/authentication-resources/certificate-signing-request-v1/) 
+using the [Kubernetes API](#signer-api), the signer's identity may be given [privileges](#authorization)
+over the entire domain by using trailing wild-cards in the `resourceNames`:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: top-level-csr-approver
+rules:
+- apiGroups:     ["certificates.k8s.io"]
+  resourceNames: ["example.com/*"]
+  resources:     ["signers"]
+  verbs:         ["approve", "sign"]
+```
+
+As `resourceNames` are opaque to the RBAC authorizer, however, an entity with permission
+to approve and sign for an entire domain cannot delegate specific paths to other entities.
+Applications which install signers therefore require `approve` and `sign` permissions for all
+signers, which is an over-broad permissions set and leads to a poor security posture as the
+credentials for such installers are good targets for actors wishing to escalate their privileges.
+
+On clusters with [ValidatingAdmissionPolicies](/docs/reference/access-authn-authz/validating-admission-policy/),
+these installers may have their privileges restricted to the minimal set required for their function.
+
+### Set up a service account for the installer
+
+In this example, we will assume some application that installs CSR signers on a Kubernetes cluster.
```

Comment:
Per the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) we [avoid using “we“](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we).